taskcluster/app_services_taskgraph/__init__.py

```diff
@@ -25,7 +29,14 @@ def _import_modules(modules):
 
 
 def get_decision_parameters(graph_config, parameters):
-    if parameters["tasks_for"] == "github-pull-request":
+    parameters["head_tag"] = None
```

Comment:
Note to self: this might break if I remove the `try_task_config.json`.

Reply:
Nit: I don't think you need to unset this and re-read it from the environment. I changed taskgraph to support this case in [bug 1589684](https://bugzilla.mozilla.org/show_bug.cgi?id=1589684). Example usage: https://github.com/mozilla-mobile/fenix/pull/6116

Reply:
Ah, neat, I already had the changes from `.taskcluster.yml` but I fixed it to resemble Fenix in `__init__.py` as well. Thanks!